docs/verifiable-document/wrapping-document.md

```diff
@@ -12,7 +12,7 @@ In this guide, we will learn how to generate the checksum by running the `wrappi
 
 ## Prerequisites
 
-- [OpenAttestation CLI](../component/open-attestation-cli) installed
+- [OpenAttestation CLI](../../component/open-attestation-cli) installed
```

Comment:
this too: https://deploy-preview-33--laughing-cori-52b566.netlify.app/docs/verifiable-document/wrapping-document#prerequisites

Reply:
done